sdk/ai/azure-ai-projects/README.md

```diff
@@ -39,6 +39,232 @@ Use the returned token credential to authenticate the client:
 
 ## Examples
 
+### Agents
+
+The following steps outline the typical sequence for interacting with agents:
+
+1. Create a project client.
+2. Upload files and associate them with a vector store for file search.
+3. Create a thread.
+4. Create a message.
+5. Execute `Run`, `Run_and_Process`, or `Stream`.
+6. Clean up by deleting resources.
+
+Additionally, you can retrieve messages.
+
+#### Create Project Client
+
+To create a project client, use the following example:
+
+<!-- SNIPPET:sample_agents_basics.create_project_client -->
+
+```python
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+```
+
+<!-- END SNIPPET -->
+
+#### Create Vector Store and Upload File
+
+To enable agents to search your files, upload the files and associate them with a vector store:
+
+<!-- SNIPPET:sample_agents_file_search.upload_file_and_create_vector_store -->
+
+```python
+openai_file = project_client.agents.upload_file_and_poll(file_path="product_info_1.md", purpose="assistants")
+openai_vectorstore = project_client.agents.create_vector_store_and_poll(
+    file_ids=[openai_file.id], name="my_vectorstore"
+)
+```
+
+<!-- END SNIPPET -->
+
+Note: If the file is for code interpretation and not for file search, a vector store is not needed.
+
+### Create Agent
+
+To create an agent with file search capabilities, use the following example:
```

Comment:
Can you explain what the developer needs to change in this sample code to fit their needs? For example, can you say a few words on where the developer will find the value of "model"? Is this the model deployment name in the Azure OpenAI connection, in the AI Studio? What does "name" mean? can the developer put anything they want there?

Reply:
I would stay away document anything related to portal.  They are going to change the UX, we can't trace them or re-publish readme when they have new changes although the SDK.    But you raise a good question that I don't know how to resolve yet.